walkthroughs/week-3-server/examples/todo-list/src/main/java/com/google/sps/servlets/NewTaskServlet.java

```diff
@@ -38,7 +38,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(taskEntity);
-
+    
```

Comment:
extra space
```suggestion
```

Reply:
Please address

It looks like the only change in this file is adding this space. Please remove since it's not actually adding anything.

you can use `git diff` to look at the changes you're committing.